src/cpp/include/openvino/genai/generation_config.hpp

```diff
@@ -31,9 +31,10 @@ enum class StopCriteria { EARLY, HEURISTIC, NEVER };
  * be used while greedy and beam search parameters will not affect decoding at all.
  *
  * Generic parameters:
- * @param max_length the maximum length the generated tokens can have. Corresponds to the length of the input prompt +
+ * @param max_length the maximum length of generated tokens for each processed audio chunk (30 seconds). Corresponds to the length of the input prompt +
  *        `max_new_tokens`. Its effect is overridden by `max_new_tokens`, if also set.
- * @param max_new_tokens the maximum numbers of tokens to generate, excluding the number of tokens in the prompt. max_new_tokens has priority over max_length.
+ * @param max_new_tokens the maximum numbers of tokens to generate for each processed audio chunk (30 seconds), excluding the number of tokens in the prompt.
```

Comment:
it's generic doc. Why do we mention chunks here? maybe we can add a note instead?

Reply:
My bad, I mixed it with Whisper config. Not sure where to add a whisper specific note. Maybe we can skip it